app/components/works/edit/deposit_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<p>
+  Be sure you have completed all the required sections. Your deposit may take several hours
+  to complete if your filesize is larger than GB.
```

Comment:
Intentional to leave the # of GB blank for now?

Reply:
Wasn't sure what to put there in MVP0, so left it blank. But I could put a placeholder value? I'll put 10GB there for now, following what we have with H2.